tools/devtool

```diff
@@ -1507,10 +1518,46 @@ cmd_release() {
             post_release "$version" "$owner" "$repo" "$token" "$prerelease" "$release_text" ||
             delete_remote_tag "$version" "$remote" "Could not post release to GitHub."
             say "Draft release created successful. Go to https://github.com/$owner/$repo/releases to check it out!"
+
+            say "Updating \`latest\` release version tag..."
+            update_latest_tag "$version" "$remote" ||
+            delete_remote_tag "$version" "$remote" "Could not update latest tag."
```

Comment:
By this point, the release is already posted to Github and it has a remote version tag. I think we should keep that tag even if for whatever reason the "latest" tag was not updated.
@luminitavoicu what do you think?

Reply:
Done.